Bug: portalClient.redirectUris/webOrigins não cobrem os 3 subdomínios das SPAs Uni+ (standalone)

## Contexto

PR #171 (story #162) entregou os 3 frontends Angular nos subdomínios:
- portal.standalone.portaluni.com.br
- selecao.standalone.portaluni.com.br
- ingresso.standalone.portaluni.com.br

Eles compartilham o mesmo Keycloak client público \`uniplus-portal\` no realm \`uniplus\` (1 client, 3 redirectUris — pattern documentado).

O smoke E2E browser logo após o merge do PR #171 expôs:

\`\`\`
[ERROR] Framing 'https://standalone.portaluni.com.br/' violates frame-ancestors 'self'
[ERROR] (Keycloak) silent-check-sso → 400
[Keycloak HTML] Parâmetro inválido: redirect_uri
\`\`\`

Resultado: Angular APP_INITIALIZER trava no \`keycloak-js.init\`, UI não monta (tela branca em todas as 3 SPAs).

## Causa raiz

\`environments/standalone/values.yaml\` na chave \`keycloak.realmImport.portalClient\`:

\`\`\`yaml
portalClient:
  clientId: uniplus-portal
  rootUrl: https://standalone.portaluni.com.br
  redirectUris:
    - https://standalone.portaluni.com.br/*   # ❌ só este
  webOrigins:
    - "+"  # deriva de redirectUris → também só standalone.portaluni.com.br
\`\`\`

Os 3 subdomínios criados em #162 não foram propagados para o realm-import.

## Fix

Adicionar os 3 subdomínios à lista \`redirectUris\`. \`webOrigins: ["+"]\` então deriva automaticamente as Web Origins corretas (3 entradas adicionais), o que também desbloqueia frame-ancestors do silent-check-sso.

\`\`\`yaml
redirectUris:
  - https://standalone.portaluni.com.br/*
  - https://portal.standalone.portaluni.com.br/*
  - https://selecao.standalone.portaluni.com.br/*
  - https://ingresso.standalone.portaluni.com.br/*
\`\`\`

## Caveat operacional (documentar no PR)

\`--import-realm\` em Keycloak 26.x é **idempotente** — não re-importa realm existente (preserva edits via Admin UI). Aplicar fix em values.yaml é a fonte de verdade durável, mas o realm já em runtime precisa ser patched via Admin API para o efeito ser imediato (passo manual descrito no PR + RUNBOOK).

Drift management automatizado de realm (\`keycloak-config-cli\` ou similar) é um follow-up separado, fora do escopo deste fix.

## Critérios de aceite

- [ ] \`environments/standalone/values.yaml\` lista os 3 subdomínios em \`portalClient.redirectUris\`
- [ ] \`helm template apps/keycloak-replica -f environments/standalone/values.yaml\` renderiza ConfigMap com os 4 redirectUris
- [ ] Patch via Admin API aplicado no realm ativo
- [ ] Smoke E2E: \`https://portal.standalone.portaluni.com.br/\` carrega Angular sem erro CSP frame-ancestors / 400 silent-check-sso
- [ ] Idem selecao + ingresso

## Refs

- PR #171 (mergeado, entregou os 3 subdomínios)
- Story #162 (entregou chart uniplus-web)
- Epic #40 (standalone OCI)

--- a/environments/standalone/values.yaml
+++ b/environments/standalone/values.yaml
@@ -391,8 +391,18 @@ keycloak:
     portalClient:
       clientId: uniplus-portal
       rootUrl: https://standalone.portaluni.com.br
+      # Cliente público compartilhado pelas 3 SPAs Uni+ (1 client, 3
+      # redirectUris) — pattern definido na story uniplus-infra#162. Cada
+      # SPA roda no próprio subdomain (PR #171); o silent-check-sso do
+      # keycloak-js exige que cada origem esteja autorizada explicitamente.
       redirectUris:
         - https://standalone.portaluni.com.br/*
+        - https://portal.standalone.portaluni.com.br/*
+        - https://selecao.standalone.portaluni.com.br/*
+        - https://ingresso.standalone.portaluni.com.br/*
+      # "+" deriva Web Origins automaticamente dos redirectUris acima
+      # (Keycloak preenche cada host como origem permitida). Isso libera
+      # tanto CORS quanto frame-ancestors para o iframe silent-check-sso.
       webOrigins:
         - "+"
 

